### PR TITLE
Add GitHub Actions

### PR DESCRIPTION
**Pull request by micka-fdz**
Sources:

- https://github.com/PrestaShop/productcomments/tree/dev/.github
- https://github.com/PrestaShop/classic-theme/tree/develop/.github

### CODE REVIEW

**Review by okom3pom — APPROVED** [review]

File: .github/workflows/build-release.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+name: Build
+on: [push, pull_request]
+
+jobs:
+  deploy:
+    name: Build dependencies & create artifact
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.0.0
```
Comment (PrestaEdit) [inline .github/workflows/build-release.yml:10]: Il me semble qu'il faille désormais utiliser les v3 des actions génériques, sous Node 16.

**Review by nicosomb — COMMENTED** [review]
Why do you commit vendor and build files?

**Comment by micka-fdz** [comment]
@nicosomb I just removed these folders from git.

**Comment by nicosomb** [comment]
@micka-fdz oh yes sorry! 👍

File: .github/workflows/build-release.yml (hunk 1 above)
Comment (micka-fdz) [reply .github/workflows/build-release.yml]: Hi @PrestaEdit, I think you're right. Thanks 🙏

**Pull request merged**